config/kernel/linux-rockchip64-edge.config

```diff
@@ -3479,13 +3819,13 @@ CONFIG_WLCORE_SDIO=m
 CONFIG_WILINK_PLATFORM_DATA=y
 CONFIG_RTL8723DU=m
 CONFIG_RTL8723DS=m
-CONFIG_RTL8822CS=m
-CONFIG_RTL8822BU=m
-CONFIG_RTL8821CU=m
-CONFIG_88XXAU=m
-CONFIG_RTL8192EU=m
-CONFIG_RTL8189FS=m
-CONFIG_RTL8189ES=m
+# CONFIG_RTL8822CS is not set
```

Comment:
Why got those disabled? I guess some USB WiFi adapters have these chips build into?

Reply:
RTL8822CS - SDIO version of RTL8822 chipset.  it is not used in usb sticks